frontend/sw360-portlet/src/main/resources/META-INF/resources/html/projects/includes/attachmentUsages.jspf

```diff
@@ -38,12 +38,18 @@
             <core_rt:if test="${projectList.size() > 1 or (projectList.size() == 1 and not empty projectList.get(0).linkedReleases)}">
                 (<a href="#" id="expandAll" class="text-primary"><liferay-ui:message key="expand.all" /> </a>|
                 <a href="#" id="collapseAll" class="text-primary"> <liferay-ui:message key="collapse.all" /></a>)
+                <select class="mx-2" id="releaseFilter">
```

Comment:
Would it be possible to use bootstrap class `form-control` like it is used in other places like obligation etc.
So that dropdown looks similar in entire sw360

Reply:
incorporated the changes.